src/Model/SmimeEmail.php

```diff
@@ -0,0 +1,202 @@
+<?php
+
+namespace SilverStripe\SmimeForms\Model;
+
+use Psr\Container\NotFoundExceptionInterface;
+use ReflectionClass;
+use ReflectionException;
+use SilverStripe\Control\Email\Email;
+use SilverStripe\Core\Injector\Injector;
+use Symfony\Component\Mailer\Exception\TransportExceptionInterface;
+use Symfony\Component\Mailer\MailerInterface;
+use Symfony\Component\Mime\Crypto\SMimeEncrypter;
+use Symfony\Component\Mime\Crypto\SMimeSigner;
+
+/**
+ * Used to override Silverstripe's Email class to enable digital signing
+ * and encryption of emails before sending them with Symfony's Mailer class.
+ */
+class SmimeEmail extends Email
```

Comment:
CORE: Change of approach from `SMimeMailer` to `SMimeEmail`. Still manipulates the `send` function for _signing_ and _encryption_ however this is now done at an `Email` level rather than at a `Mailer` level